### PR TITLE
test(test): add tests for console user management

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Add integration tests for user management in admin console

## Testing

Integration Tests

image

## Checklist

- [x] integration tests

### CODE REVIEW

File: packages/integration-tests/screenshots/confirm-dialog.png
Comment (wangsijie) [inline packages/integration-tests/screenshots/confirm-dialog.png:1]: Is this file been checked in on purpose?

File: packages/integration-tests/package.json [hunk 1]
```diff
@@ -21,10 +21,11 @@
     "start": "pnpm test"
   },
   "devDependencies": {
+    "@jest/test-sequencer": "^29.5.0",
     "@jest/types": "^29.1.2",
     "@logto/connector-kit": "workspace:^1.1.0",
-    "@logto/language-kit": "workspace:^1.0.0",
     "@logto/js": "^2.0.1",
+    "@logto/language-kit": "workspace:^1.0.0",
```
Comment (wangsijie) [inline packages/integration-tests/package.json:28]: unused?

**Review by wangsijie — APPROVED** [review]
Overall LGTM

File: packages/integration-tests/package.json (hunk 1 above)
Comment (charIeszhao) [reply packages/integration-tests/package.json:28]: Hmmm, seems so.
But I didn't add it myself, it's just re-ordered. Let me try to remove it.

File: packages/integration-tests/screenshots/confirm-dialog.png
Comment (charIeszhao) [reply packages/integration-tests/screenshots/confirm-dialog.png:1]: Oops... no, this is a testing screenshot. Will remove it

**Pull request merged**